tests-system/lobster-python/rbt-output-file/default-scenario/expected-output/stdout.txt

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Please use `lobster-python --out=basic.lobster` such that it creates a dedicated output file.
The system tests shall consist of (at least) two files for the expected output:
- `stdout.txt` for the expected command line output
- `basic.lobster` for the expected lobster file